llvm/lib/Target/RISCV/RISCVCallingConv.td

```diff
@@ -18,6 +18,9 @@ def CSR_ILP32E_LP64E : CalleeSavedRegs<(add X1, X8, X9)>;
 def CSR_ILP32_LP64
     : CalleeSavedRegs<(add CSR_ILP32E_LP64E, (sequence "X%u", 18, 27))>;
 
+def CSR_RISCV_Hotspot_JIT
+    : CalleeSavedRegs<(add CSR_ILP32_LP64)>;
```

Comment:
Do we need X18-X27 as callee-saved regs for hotspot? Register save types in C2 are SOC (Save-On-Call) or NS (No-Save).

https://github.com/jeandle/jeandle-jdk/blob/9533cb77d6ed4a81054e6bcd363dde0238aaf270/src/hotspot/cpu/riscv/riscv.ad#L69-L78